Mod2/Lessons/Week2/GitForPairs.md

```diff
@@ -0,0 +1,132 @@
+---
+layout:     page
+title:      Git for Pairs
+permalink:  git-for-pairs
+---
+
+# Git for Pairs
+
+## Learning Goals
+  * Outline a workflow for collaborating on a single repository
+  * Identify and resolve merge conflicts
+
+## Vocabulary
+  * Pull Request (PR)
+  * Clone repo
+  * Fork repo
+
+## Review
+> If necessary, review the following video lessons from Version Control with Git/Github
```

Comment:
As a student, how will I know that I need to review these lessons?  This might be great prep work for the lesson - I don't think it will hurt for all students to do this review.

Reply:
Good call.